Topology/README.md

```diff
@@ -23,15 +23,15 @@ The following are optional fields and default values (option, more a list of pos
   - `comment` (str) - Any additional comment one would attach to the molecule.
```

Comment:
I think, based on #35 that it makes sense to have an "identifier" section, with comment, molecule name, formula, InChI, SMILES, etc. as optional.  This would probably also include provenance and DOI.

Looking at my notes, I think these were grouped in the discussion.

Reply:
Are you thinking of `comment` as a dictionary? I was considering moving up a few of those to top level optional fields.

Reply:
No, I'm thinking that there's an obvious request for identifiers, and I think comment/title have usually been identifiers in QC codes.

So my suggestion is that there's an explicit identifier object - something like:

```
"identifier" : {
   "name": "aspirin",
   "comment": "training set",
   "formula": "C9H8O4",
   "smiles": "O=C(C)Oc1ccccc1C(=O)O",
   "inchikey": "BSYNRYMUTXBXSQ-UHFFFAOYSA-N"
}
```